cns/service/main.go

```diff
@@ -690,13 +694,15 @@ func main() {
 	// Log platform information.
 	logger.Printf("Running on %v", platform.GetOSInfo())
 
+	storeFileLocation = "./azure-cns.json"
 	err = platform.CreateDirectory(storeFileLocation)
 	if err != nil {
 		logger.Errorf("Failed to create File Store directory %s, due to Error:%v", storeFileLocation, err.Error())
 		return
 	}
 
-	lockclient, err := processlock.NewFileLock(platform.CNILockPath + name + store.LockExtension)
+	lockPath := "./"
+	lockclient, err := processlock.NewFileLock(lockPath + name + store.LockExtension)
```

Comment:
To make it easier for reviewers, it would be helpful if you added a quick comment stating that these changes are temporary and won't be merged

Reply:
```suggestion
	// DO NOT MERGE
	lockPath := "./"
	lockclient, err := processlock.NewFileLock(lockPath + name + store.LockExtension)
```

Reply:
I added that comment on every line that is supposed to be temporary and not meant to be merged